src/main/resources/patch_003.txt

```diff
@@ -0,0 +1 @@
+ALTER TABLE cg_link ADD subtype VARCHAR(50);
```

Comment:
Also, I've updated wiki in my fork according to changes.
Will copy paste it to Devoted once update will be released to the server